share/extender/build_input.yml

```diff
@@ -76,6 +76,7 @@ platforms:
 
   common:
     env:
+      ZIG_PATH:      '{{env.ZIG_PATH_0_11}}'
```

Comment:
Does this mean that we now support zig files as well? We should make a note of that in the PR notes 👍

Reply:
Unfortunately, I think there was an issue with the extender server (for some reason), so I didn't pursue it more.
Yeah, this PR is _really_ old (remember the Zig presentation?), and it was intended to land in a first iteration a long time ago.